### ISSUE TITLE
Download custom DB

### ISSUE THREAD

**Issue by spabinger** [pirovc/genome_updater#1]
Hi,

while trying to download a custom database I get the following error:

* Command:
```
./genome_updater.sh -d "refseq" -g "fungi,viral" -f "genomic.fna.gz,genomic.gbff.gz" -t 12 -o fungi_viral_genomes/

```
* Error:
```
/bin/bash: line 4:
                        wget --no-verbose --continue ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/477/545/GCF_001477545.1_Pneu_cari_B80_V3/GCF_001477545.1_Pneu_cari_B80_V3_genomic.fna.gz --tries=5 --read-timeout=20 --append-output=fungi_viral_genomes//2017-09-11_17-03-58.log -P fungi_viral_genomes//files/;
                        if [ 0 -eq 1 ]; then check_md5_ftp "ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/477/545/GCF_001477545.1_Pneu_cari_B80_V3/GCF_001477545.1_Pneu_cari_B80_V3_genomic.fna.gz"; fi;
                        print_progress "1" 30906
                : No such file or directory
```

<br/>
- wget alone works.

Thanks for your help,
Stephan

**Comment by pirovc** [comment]
Hi Stephan,

The same command works here. It's probably some incompability with some shell tool. 

Do you have parallel installed? If yes, please send me the result for the following commands so I can check what's wrong.

`bash --version`
`parallel --version`
`awk --version`
`wget --version`
`join --version`
`bc --version`

Best,
Vitor

**Comment by spabinger** [comment]
Hi,

thanks for the fast reply:

```
bash --version
GNU bash, version 4.3.11(1)-release (x86_64-pc-linux-gnu)

parallel --version
WARNING: YOU ARE USING --tollef. IF THINGS ARE ACTING WEIRD USE --gnu.
GNU parallel 20121122

awk --version
GNU Awk 4.0.1

wget --version
GNU Wget 1.15 built on linux-gnu.

join --version
join (GNU coreutils) 8.21

bc --version
bc 1.06.95
```

Best,
Stephan

**Comment by pirovc** [comment]
Hi Stephan,

Nice when the version command helps to solve the problem :) "WARNING: YOU ARE USING --tollef. IF THINGS ARE ACTING WEIRD USE --gnu." I wish it was always like this.

I implemented it and the new version is online. Please clone it again or `git pull origin` inside the genome_updater folder.

Let me know if that solved the problem.

Best,
Vitor

**Comment by spabinger** [comment]
Hi,

I could successfully download the files. Thanks!

Best,
Stephan

**Comment by pirovc** [comment]
Solved #2
